railties/lib/rails/test_unit/runner.rb

```diff
@@ -48,7 +57,17 @@ def run(argv = [])
         def load_tests(argv)
           patterns = extract_filters(argv)
           tests = list_tests(patterns)
-          tests.to_a.each { |path| require File.expand_path(path) }
+          tests.to_a.each do |path|
+            require File.expand_path(path)
+          rescue LoadError => exception
+            ms = list_tests([default_test_glob])
+            cs = DidYouMean::SpellChecker.new(dictionary: ms).correct(path)
```

Comment:
Are `ms` and `cs` short for "misspelling" and "correct spelling"? Would replacing the abbreviations with long form words lose us anything?

Reply:
I'm not sure the original intention, so good call.

I went with `all_tests` and `corrections`, since the former is all of the tests using the default glob and corrections is obvious.